Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM lsiobase/alpine.python:3.6
+MAINTAINER sparklyballs
+
+# set version label
+ARG BUILD_DATE
+ARG VERSION
+LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
+
+# install packages
+RUN \
+ apk add --no-cache \
+	--repository http://nl.alpinelinux.org/alpine/edge/community \
+	mediainfo gdbm
+
+# install app
+COPY . /app/medusa/
+
+# copy local files
+COPY docker/root/ /
+
+# ports and volumes
+EXPOSE 8081
+VOLUME /config /downloads /tv
```

Comment:
Would you mind adding a volume mount for anime since some people do like to keep them separate.